src/manual/Installing/Installing-from-source-Unix.md

```diff
@@ -48,9 +48,14 @@ apk update && apk add \
   linux-headers ncurses-dev openssl-dev pcre2-dev zlib-dev
 ```
 
+Version 0.105+: install the Rust toolchain. The best option is to [install the Rut toolchain using rustup](#install-rust-toolchain) your Rust toolchain. Alpine users on the latest release may also find an adequate Rust toolchain with:
```

Comment:
This should be "install the Rust toolchain" not "Rut"

Reply:
Fixed